### PR TITLE
Add SuiteCRM To EasyPanel!

### PR DESCRIPTION
**Pull request by DrMxrcy**

### CODE REVIEW

File: templates/suitecrm/index.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  randomPassword,
+  randomString,
+  Services,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const mariaPassword = randomPassword();
+  const appKey = Buffer.from(randomString(32)).toString("base64");
```
Comment (deiucanta) [inline templates/suitecrm/index.ts:12]: Isn't `randomString` enough here?

File: templates/suitecrm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: SuiteCRM
+description: SuiteCRM is a completely open source, enterprise-grade Customer Relationship Management (CRM) application.
+instructions: null
+changeLog:
+  - date: 2023-1-12
+    description: first release
+links:
+  - label: Github
+    url: https://github.com/salesagility/SuiteCRM
```
Comment (deiucanta) [inline templates/suitecrm/meta.yaml:9]: Can you please add the website? https://suitecrm.com/. You can add the docs as well: https://docs.suitecrm.com/

File: templates/suitecrm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      title: App Service Name
+      default: suitecrm
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: bitnami/suitecrm:8
+    databaseServiceName:
+      type: string
+      title: Database Service Name
+      default: suitecrm-roadmap-db
```
Comment (deiucanta) [inline templates/suitecrm/meta.yaml:35]: Can we make this name simply `suitecrm-db`?

**Comment by deiucanta** [comment]
Here is a better logo

File: templates/suitecrm/index.ts (hunk 1 above)
Comment (Supernova3339) [reply templates/suitecrm/index.ts:12]: Checked myself, needs to be this specific to work

**Pull request merged**